src/wireviz/DataClasses.py

```diff
@@ -95,8 +95,8 @@ class Connector:
     pincount: Optional[int] = None
     image: Optional[Image] = None
     notes: Optional[MultilineHypertext] = None
-    pinlabels: List[Pin] = field(default_factory=list)
     pins: List[Pin] = field(default_factory=list)
+    pinlabels: List[Pin] = field(default_factory=list)
```

Comment:
I considered that, since it's an optional attribute, I would put it further down. But you are right, there is no preferred order of attributes at the moment...